src/logic/safe/store/actions/transactions/fetchTransactions/loadGatewayTransactions.ts

```diff
@@ -42,25 +42,15 @@ export const loadPagedHistoryTransactions = async (
 export const loadHistoryTransactions = async (safeAddress: string): Promise<HistoryGatewayResponse['results']> => {
   const historyTransactionsUrl = getHistoryTransactionsUrl(safeAddress)
 
-  try {
-    const {
-      data: { results, ...pointers },
-    } = await axios.get<HistoryGatewayResponse, AxiosResponse<HistoryGatewayResponse>>(historyTransactionsUrl)
-
-    if (!historyPointers[safeAddress]) {
-      historyPointers[safeAddress] = pointers
-    }
-
-    return results
-  } catch (error) {
-    // When the safe is just created there is a delay until the gateway recognize the
-    // safe address, when that happens it returns 404.
-    if (error.response.status === 404) {
```

Comment:
This was failing when `error.response` was undefined (e.g. when network is offline).